modules/nw-using-cookies-keep-route-statefulness.adoc

```diff
@@ -19,3 +19,10 @@ for the session. The cookie is passed back in the response to the request and
 the user sends the cookie back with the next request in the session. The cookie
 tells the Ingress Controller which endpoint is handling the session, ensuring
 that client requests use the cookie so that they are routed to the same pod.
+
+[NOTE]
+====
+Cookies cannot be set on passthrough routes, because the HTTP traffic cannot be seen. Instead, a number is calculated based on the source IP address, which determines the back-end.
```

Comment:
```suggestion
Cookies cannot be set on passthrough routes, because the HTTP traffic cannot be seen. Instead, a number is calculated based on the source IP address, which determines the back end.
```

When using back end as a noun it should not be hyphenated. I typically use "backend" as a closed compound word to avoid trying to determine whether I need to use back-end vs. back end.  I would choose either "backend" or "back end" in this context. 

For your reference, IBM Style Guide has some guidance on "backend vs. back-end vs. back end" under Word Usage.